lib/orchestrator_client.rb

```diff
@@ -55,7 +55,7 @@ def post(location, body)
     req.body = body.to_json
     res = https.request(req)
 
-    if res.code != "202"
+    unless Set.new(["202", "200"]).include? res.code
```

Comment:
Using a one-time set here brings no actual benefit.

Reply:
what is the other ruby way to do this?

Reply:
oh for some reason I thought arrays responded to include based on their indexes

Reply:
https://ruby-doc.org/core-2.2.0/Array.html#method-i-include-3F